packages/flutter_tools/bin/xcode_backend.sh

```diff
@@ -114,7 +119,8 @@ BuildApp() {
     fi
 
     StreamOutput " ├─Building Dart code..."
-    RunCommand "${FLUTTER_ROOT}/bin/flutter" --suppress-analytics build aot \
+    RunCommand "${FLUTTER_ROOT}/bin/flutter" --suppress-analytics           \
+      ${verbose_flag} build aot                                             \
```

Comment:
nit: can we do one per line like

verbose_flag
build aot

here and below?

Reply:
I like it! Done.